chatwoot/chatwoot
# feat: Notify the user that new conversations are created if on a folder or filter view

# Pull Request Template

## Description

This PR adds the ability to notify the user that new conversations are created if on a folders or filters view

Fixes #3859 

## Type of change


## How Has This Been Tested?

1. Create a filter or save a folder.
2. Go to the filter or saved folder view.
3. Now you can see a banner when a new conversation is created.

**Screenrecording**


https://user-images.githubusercontent.com/64252451/152743492-944472e5-38de-451f-b8a2-f0ff70942ba7.mov



## Checklist:
